reference/configuration.rst

```diff
@@ -49,6 +49,16 @@ This value defaults to true, and is effective on the coordinator. In rare cases,
 complex upgrade processes may require setting this parameter to false, thus
 disabling the check.
 
+citus.log_distributed_deadlock_detection (boolean)
```

Comment:
These look good, but I might still add `max_task_string_size`. We usually talk about the type of the param, min/max values etc, which might be useful.